oauth-authorization-server/src/main/java/com/baeldung/config/OAuth2AuthorizationServerConfigJwt.java

```diff
@@ -88,8 +92,8 @@ public TokenEnhancer tokenEnhancer() {
         return new CustomTokenEnhancer();
     }
 
-    //@Bean
-    //public BCryptPasswordEncoder passwordEncoder() {
-    //    return new BCryptPasswordEncoder();
-   // }
+   @Bean
```

Comment:
I think this bean was defined already under one of the configuration files. (That's why I commented it out) It's autowired here. Please double check and delete this bean definition.

Reply:
The autowiring would not catch up unless it was not commented